packages/gestalt/src/SelectList.js

```diff
@@ -36,147 +36,128 @@ type Props = {|
-    this.select = ref;
-  };
+export default function SelectList({
+  disabled = false,
+  errorMessage,
+  helperText,
+  id,
+  label,
+  name,
+  onChange,
+  options = [],
+  placeholder,
+  size = 'md',
+  value,
+}: Props): Node {
+  const [focused, setFocused] = useState(false);
 
-  handleOnChange: (
+  const handleOnChange: (
     event: SyntheticInputEvent<HTMLSelectElement>
   ) => void = event => {
-    const { onChange, value } = this.props;
     if (value !== event.target.value) {
       onChange({ event, value: event.target.value });
     }
   };
 
-  render(): Node {
-    const {
-      disabled,
-      errorMessage,
-      helperText,
-      id,
-      label,
-      name,
-      options,
-      placeholder,
-      size,
-      value,
-    } = this.props;
+  const handleBlur = (event: SyntheticInputEvent<HTMLSelectElement>) => {
+    setFocused(false);
+    handleOnChange(event);
+  };
 
-    const { focused } = this.state;
+  const handleFocus = () => {
+    setFocused(true);
+  };
 
-    const classes = classnames(
-      styles.select,
-      formElement.base,
-      disabled ? formElement.disabled : formElement.enabled,
-      errorMessage ? formElement.errored : formElement.normal,
-      size === 'md' ? layout.medium : layout.large
-    );
+  const classes = classnames(
+    styles.select,
+    formElement.base,
+    disabled ? formElement.disabled : formElement.enabled,
+    errorMessage ? formElement.errored : formElement.normal,
+    size === 'md' ? layout.medium : layout.large
+  );
 
-    const showPlaceholder = placeholder && !value;
+  const showPlaceholder = placeholder && !value;
 
-    return (
-      <Box>
-        {label && <FormLabel id={id} label={label} />}
+  return (
+    <Box>
+      {label && <FormLabel id={id} label={label} />}
+      <Box
+        color={disabled ? 'lightGray' : 'white'}
+        display="flex"
+        position="relative"
+        rounding={4}
+        width="100%"
+      >
         <Box
-          color={disabled ? 'lightGray' : 'white'}
+          alignItems="center"
+          bottom
+          dangerouslySetInlineStyle={{
+            __style: { paddingRight: 14, paddingTop: 2 },
+          }}
           display="flex"
-          position="relative"
-          rounding={4}
-          width="100%"
+          position="absolute"
+          right
+          top
         >
-          <Box
-            alignItems="center"
-            bottom
-            dangerouslySetInlineStyle={{
-              __style: { paddingRight: 14, paddingTop: 2 },
-            }}
-            display="flex"
-            position="absolute"
-            right
-            top
-          >
-            <Icon
-              icon="arrow-down"
-              size={12}
-              color={disabled ? 'gray' : 'darkGray'}
-              accessibilityLabel=""
-            />
-          </Box>
-          <select
-            aria-describedby={errorMessage && focused ? `${id}-error` : null}
```

Comment:
Previously `focused` was never set to `true`, which was a bug. So we added `onFocus`